morpheus/_lib/src/stages/file_source.cpp

```diff
@@ -114,54 +92,6 @@ FileSourceStage::subscriber_fn_t FileSourceStage::build()
     };
 }
 
-cudf::io::table_with_metadata FileSourceStage::load_table()
-{
-    auto file_path = std::filesystem::path(m_filename);
-
-    if (file_path.extension() == ".json" || file_path.extension() == ".jsonlines")
-    {
-        // First, load the file into json
-        auto options = cudf::io::json_reader_options::builder(cudf::io::source_info{m_filename}).lines(true);
-
-        auto tbl = cudf::io::read_json(options.build());
```

Comment:
This is a good catch. Was going to suggest you do this as well but you beat me to it.